pkg/apis/rook.io/v1/types.go

```diff
@@ -40,28 +41,30 @@ type Node struct {
 	Name      string                  `json:"name,omitempty"`
 	Resources v1.ResourceRequirements `json:"resources,omitempty"`
 	Config    map[string]string       `json:"config"`
-	Selection
+	Selection SelectionSpec           `json:"selection"`
```

Comment:
Changing this to be a separate field instead the inline struct will change a lot of places. Was this necessary for the generator? Seems like this would change the structure of the yaml and break backward compatibility.

Reply:
Fixed differently by using the right json tag: `json:",inline"`